android/titanium/src/java/org/appcelerator/titanium/proxy/ActionBarProxy.java

```diff
@@ -378,6 +391,11 @@ private Drawable getDrawableFromUrl(String url)
 		return tfh.loadDrawable(imageUrl.resolve(), false);
 	}
 
+	private void handleSetCustomView(TiViewProxy view)
+	{
+		actionBar.setCustomView(view.getOrCreateView().getNativeView());
```

Comment:
We need to handle the case where the custom view is set to `null`.

Perhaps do one of the following:
1. Ignore null and no-op. (Simple case.)
2. Ideally, remove the last custom view set, but I don't think ActionBar supports this. Calling `setDisplayShowCustomEnabled(false)` would hide it, but still exist in memory.

Also, we may want to explicitly call `setDisplayShowCustomEnabled(true)` just in case a forked Android OS doesn't have this as the default setting.

Reply:
The Android method `setCustomView()` accepts `null` as a parameter and it removes the latest custom view added. I think this is fine. We get the Android native view through the proxy that is passed and it will be released once it is not referenced anymore.
`setDisplayShowCustomEnabled()` is called with true the first time a custom view is used.